pegasus/sites.v3/code.org/views/minecraft_2018.haml

```diff
@@ -1,5 +1,21 @@
 %link{href: "/css/minecraft2018.css", rel: "stylesheet"}
 
+:ruby
+  tutorial = {
+    heading:  I18n.t(:hoc2018_mc_educators),
+    background_image: "/images/fill-1940x700/mc/minecraft_teacher_background.jpg",
```

Comment:
Should background_image be moved to the minecraft css file?  Then it will be easier to change the height/width if ever required.